example_configs/dropwizard-0-8-x.yml

```diff
@@ -0,0 +1,153 @@
+---
+lowercaseOutputLabelNames: true
+lowercaseOutputName: true
+blackListObjectNames: ["metrics:type=*,name=jvm*"]
+
+rules:
+
+- pattern: "metrics<name=ch.qos.logback.core.Appender.(debug|error|info|trace|warn)><>Count"
+  name: log_msg_total
```

Comment:
All dropwizard apps use logback. It also hijacks java.util.Logging and Log4J + SLF4J CommonsLogging (and possibly others). So I see no use in pointing out that it is logback  - but can add a label if you like.